dns/image/Makefile

```diff
@@ -14,7 +14,7 @@
 
 ARCH ?= amd64
 TAG ?= 1.0
-REGISTRY ?= gcr.io/google_containers
+REGISTRY ?= k8s.gcr.io
```

Comment:
`REGISTRY ?= staging-k8s.gcr.io`

Don't know how authentication is done here, e.g. if it assumes that a cred helper is installed so that naked `docker` just works.